dev.jonmagon.kdiskmark.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: dev.jonmagon.kdiskmark
```

Comment:
as said earlier if the domain isn't registered then this doesn't meet the requirements.